### PR TITLE
[media/CircleGraph] Introduce visq mode

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce visq mode for CircleGraph to follow basic feature as default circle viewer.

ONE-vscode-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
for #1350, from #1351

File: media/CircleGraph/index.js [hunk 1]
```diff
@@ -51,6 +51,7 @@ const vscode = acquireVsCodeApi();
 const viewMode = {
     viewer: 0,
     selector: 1,
+    visq: 2,
```
Comment (dayo09) [inline media/CircleGraph/index.js:54]: Q. Could you add any documentation (README or simple comment) explaning about what 'visq' is..?

**Review by dayo09 — APPROVED** [review]

File: media/CircleGraph/index.js (hunk 1 above)
Comment (seanshpark) [reply media/CircleGraph/index.js:54]: Thanks for the comment :)
I'll add comments in another PR.

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**